docs/natspec-format.rst

```diff
@@ -36,7 +36,7 @@ for the purposes of NatSpec.
 
 -  For Vyper, use ``"""`` indented to the inner contents with bare
    comments. See `Vyper
-   documentation <https://vyper.readthedocs.io/en/latest/structure-of-a-contract.html#natspec-metadata>`__.
+   documentation <https://vyper.readthedocs.io/en/latest/natspec.html>`__.
```

Comment:
Why double instead of single underscore?

Reply:
I'm not sure but I didn't want to get too deep into small details so I left it as is.

I'll check if it works with one.

Reply:
Turns out it's a special syntax for one-off URLs that does not give it a name: [Restructured text (rst) http links underscore ('__' vs '_' use)](https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use)

I'm not sure why it's used here but there are a few other places where we have it and it does not break anything so I see no reason to remove it. Actually, maybe we should be using it more because we don't usually refer to links by names.